[DOC-228] Update documentation to reflect the new Ed-Fi Terminology Guidelines (#441)
Ed-Fi-Alliance-OSS/ed-fi-alliance-oss.github.io: docs/partners/2-certification/certification-for-data-providers/testing-requirements.md

# Testing Requirements

> The Ed-Fi Data Standard have as their goal the capture and exchange of deep

Reviewer: ```suggestion
The Ed-Fi Data Standard has as its goal the capture and exchange of deep
```
